.travis.yml

```diff
@@ -1,14 +1,17 @@
 sudo: required
-dist: trusty
+dist: xenial
```

Comment:
Ah, sorry, one more thing I overlooked, should we stick with the `trusty` container because of https://github.com/travis-ci/travis-ci/issues/5821 ?  The build logs show that it is still building on trusty anyway.